From 72c3b7324f00047e6dc5d8380ed2f6ff2494a6f9 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 18 Dec 2022 14:51:34 -0800
Subject: [PATCH] locale: Avoid using glibc specific defines on musl

musl does not provide some glibc-only enum members e.g. _NL_ADDRESS_LANG_NAME

Upstream-Status: Submitted [https://github.com/wxWidgets/wxWidgets/pull/23050]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/unix/uilocale.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/unix/uilocale.cpp b/src/unix/uilocale.cpp
index 57773e17f5..86816ba896 100644
--- a/src/unix/uilocale.cpp
+++ b/src/unix/uilocale.cpp
@@ -619,7 +619,7 @@ wxString
 wxUILocaleImplUnix::GetLocalizedName(wxLocaleName name, wxLocaleForm form) const
 {
     wxString str;
-#if defined(HAVE_LANGINFO_H) && defined(__LINUX__)
+#if defined(HAVE_LANGINFO_H) && defined(__LINUX__) && defined(__GLIBC__)
     switch (name)
     {
         case wxLOCALE_NAME_LOCALE:
